devtools/cov/rustc-proptest-fix

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
Why need this?

Reply:
> Why need this?

https://github.com/xd009642/tarpaulin/issues/161#issuecomment-455674846